### DISCUSSION TITLE
Как восстановить админский логин и пароль?

### DISCUSSION THREAD

**Discussion by alphagamma5** [YetiForceCompany/YetiForceCRM#15416] [How to] [answered]
Как восстановить админский логин и пароль?

**Comment by vovpff** [comment, marked as answer]
Check here https://github.com/YetiForceCompany/YetiForceScripts/tree/master/PasswordReset

**Comment by mariuszkrzaczkowski** [comment]
https://yetiforce.com/en/knowledge-base/documentation/developer-documentation/item/how-to-reset-a-user-s-password-without-access-to-the-crm-system
